# Waypoints.msg
std_msgs/Header header
geometry_msgs/Point hard_waypoint
geometry_msgs/Point[] soft_waypoints

